portfolio/src/main/webapp/script.js

```diff
@@ -20,18 +20,18 @@ window.onload = function(){
     console.log("I see you are looking at the console")
 }
 
-function addRandomGreeting() {
-  const facts =
-      ['I have a tabby cat named Poppy.', 'I was in a drumline.', 'I\'m part of Y Fashion House at Yale.', 'I love to paint.', 'I am from Nashville.'];
+// function addRandomGreeting() {
+//   const facts =
+//       ['I have a tabby cat named Poppy.', 'I was in a drumline.', 'I\'m part of Y Fashion House at Yale.', 'I love to paint.', 'I am from Nashville.'];
 
-  // Pick a random greeting.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
+//   // Pick a random greeting.
+//   const fact = facts[Math.floor(Math.random() * facts.length)];
 
-  // Add it to the page.
-  const factContainer = document.getElementById('message-container');
+//   // Add it to the page.
+//   const factContainer = document.getElementById('message-container');
 
-  factContainer.innerText = "You found a hidden fact! \n \n" + fact;
-}
+//   factContainer.innerText = "You found a hidden fact! \n \n" + fact;
+// }
```

Comment:
I would probably remove it rather than commenting it all out because. you can always find it in your git commit history later. Also for commenting out large chunks, I believe java lets you use something like `/* stuff */` so you don't need `//` on every line.

Reply:
Gotcha; I will remove the comments and use the other commenting format in the future.